.ci/docker-compose-file/docker-compose-rabbitmq.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.9'
+
+services:
+  rabbitmq:
+    container_name: rabbitmq
+    image: rabbitmq:3.11-management
+
+    restart: always
+    expose:
+      - "15672"
+      - "5672"
+    ports:
+      - "15672:15672"
+      - "5672:5672"
```

Comment:
Suggestion: comment those out so they don't take host ports by default, but it's easy to uncomment when developing.